### PR TITLE
Add pipeline and application usage in cli

### PR DESCRIPTION
**Pull request by codebfu**
added :
  - cdsctl pipeline usage
  - cdsctl application usage
  - sdk interface for pipeline get
  - RequestModifiers for pipeline usages
changed :
  - sdk pipeline get

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: sdk/cdsclient/interface.go [hunk 1]
```diff
@@ -457,6 +458,42 @@ func WithUsage() RequestModifier {
...
+func WithWorkflows() RequestModifier {
+	return func(r *http.Request) {
+		q := r.URL.Query()
+		q.Set("withWorkflows", "true")
+		r.URL.RawQuery = q.Encode()
+	}
+}
+
+// WithApplications allow a provider to retrieve a pipeline with its applications usage
+func WithApplications() RequestModifier {
```
Comment (richardlt) [inline sdk/cdsclient/interface.go]: Remove this request modifier and also the useless code in get pipeline handler

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12472.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12474.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12474.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Pull request merged**